chrome/updater/configurator.cc

```diff
@@ -172,9 +172,13 @@ base::flat_map<std::string, std::string> Configurator::ExtraRequestParams()
   params.insert(std::make_pair("sbversion", std::to_string(SB_API_VERSION)));
   params.insert(
       std::make_pair("jsengine", script::GetJavaScriptEngineNameAndVersion()));
+  // The flag name to force an update is changed to "is_forced_update_", but
+  // the flag name sent to Omaha remains "updaterchannelchanged" to isolate
+  // the changes to Omaha configs. If it's decided to change this flag name
+  // on Omaha, it'll be done in a separate PR after the Omaha configs are updated.
```

Comment:
nit: this sounds more like a PR comment. Maybe just mention how this mirrors "updaterchannelchanged".

Reply:
Would you suggest specifically how to rewrite this comment? Not sure if I got you right.

Reply:
Maybe "The flag name sent to Omaha is still \"updaterchannelchanged\", we use \"is_forced_update_\" to cover cases where update is requested by client but channel can stay the same"